website/src/shared/ConfigContext.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { createContext, useContext } from 'react';
+import { useAsync } from 'react-use';
+
+interface Config {
+  DISABLE_UPLOAD: boolean;
+}
+
+const ConfigContext = createContext<Config | undefined>(undefined);
+
+export const ConfigProvider = ({ children }: { children: React.ReactNode }) => {
+  const { value: config, loading, error } = useAsync(async () => {
+    const response = await fetch("/config");
```

Comment:
/config only works if backend runs on the same domain which is not always the case. Use `backendDomain`/config instead. There is some prior art if you search for it.